packages/core/src/physics/Collision.ts

```
@@ -1,5 +1,47 @@
+   * You need to obtain the actual number of contact points from the function's return value.
+   */
+  getContacts(outContacts: ContactPoint[]): number {
+    const nativeCollision = this._nativeCollision;
+    const factor = nativeCollision.shape0Id < nativeCollision.shape1Id ? 1 : -1;
+
+    const nativeContactPoints = nativeCollision.getContacts();
+    const length = nativeContactPoints.size();
+    for (let i = 0; i < length; i++) {
+      const nativeContractPoint = nativeContactPoints.get(i);
+
+      const contact = (outContacts[i] ||= new ContactPoint());
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)